documentation/source/user_section/command-line/sct_deepseg.rst

```diff
@@ -171,7 +174,7 @@ Retired models
    :target: deepseg/seg_ms_sc_mp2rage.html
 
 .. |sc_t2star| image:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/sct_deepseg/sc_t2star.png
-    :target: deepseg/sc_t2star.html
+    :target: deepseg/tar.html
```

Comment:
Likewise, this PR should not be modifying the image for `sc_t2star`.

Reply:
Fixed in [d93606c](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4832/commits/d93606cd0a5ae7788fe6c32807a1f437a66db961).